src/cli-constant.js

```diff
@@ -47,6 +50,7 @@ const detailOptions = normalizeDetailOptions({
     type: "boolean",
     category: "format",
     forwardToApi: true,
+    oppositeDescription: "Do not print spaces between brackets.",
```

Comment:
I like `oppositeDecription`.

Super small thing: Is it confusing that we have `hidden: true` but the option still shows up (although in `no-name` form)? Should the presence of `oppositeDescription` be enough to tell how the option should be displayed in `--help`?